otj-server-reactive/src/main/java/com/opentable/server/reactive/ReactiveServer.java

```diff
@@ -37,7 +37,7 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @Import({
-        DelegatingWebFluxConfiguration.class,
+        ReactiveServerJacksonConfiguration.class,
```

Comment:
@scottjohnson It is not clear why we were importing DelegatingWebFluxConfiguration here. It is imported automatically by `WebFluxAutoConfiguration` in more appropriate order.

Reply:
1. `@ReactiveServer` annotation is a strict replacement of `@EnableWebFlux` that is meant to work the same way, namely by importing ` DelegatingWebFluxConfiguration` so that app authors can continue to configure their service using the same configuration interface as is documented in Spring's guide. By removing `DelegatingWebFluxConfiguration` from this class you either A) Force developers to import two different annotations (@ReactiveServer and @EnableWebFlux instead of just @ReactiveServer) or B) Prevent developers from configuring their apps as per the Spring docs.

See [here](https://github.com/spring-projects/spring-framework/blob/e7e5cce7355f0d0224dabaa82022829f616efecf/spring-webflux/src/main/java/org/springframework/web/reactive/config/EnableWebFlux.java#L90)

If you want to fully configure your app for yourself you can import `@ReactiveServerUnconfigured` instead of `@ReactiveServer`, which skips pulling in `@DelegatingWebFluxConfiguration`.

If the order that `@ReactiveServer` imports `@DelegatingWebFluxConfiguration` is incorrect when used with `@WebFluxAutoConfiguration`, then this problem also holds true for vanilla Spring WebFlux with their `@EnableWebFlux` annotation.

2. If we are to make the other changes in this PR (which I am still reviewing), then `ReactiveServerJacksonConfiguration` should be imported in `ReactiveServerCommonConfiguration` and not in this file.